internal/provider/provider.go

```diff
@@ -373,6 +373,34 @@ func New() *schema.Provider {
 						Optional:      true,
 						ConflictsWith: []string{"command"},
 					},
+					"healthcheck_enabled": {
+						Type:          schema.TypeBool,
+						Description:   "Run a HTTP request periodically to determine the application readiness.",
+						ForceNew:      true,
+						Optional:      true,
+						ConflictsWith: []string{"command"},
+					},
+					"healthcheck_url": {
+						Type:          schema.TypeString,
+						Description:   "HTTP address used determine the application readiness.",
```

Comment:
Should document what constitutes a pass or fail. "Any 2xx response within the 5 second timeout window is considered a pass" or something.

Also add this sentence to the app.md file too.